cloudformation/bulkExport.yaml

```diff
@@ -138,3 +138,32 @@ Resources:
                 Action:
                   - s3:GetObject
                 Resource: !Join ['', [!GetAtt BulkExportResultsBucket.Arn, '/*']]
+
+  UpdateStatusLambdaRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: 'Allow'
+            Principal:
+              Service: 'lambda.amazonaws.com'
+            Action: 'sts:AssumeRole'
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+        - arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess
+      Policies:
+        - PolicyName: ddbAccess
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                  - dynamodb:UpdateItem
+                Resource:
+                  - !GetAtt ExportRequestDynamoDBTable.Arn
+              - Effect: Allow
+                Action:
+                  - kms:Decrypt

```

Comment:
When do we need KMS decryption for a DDB update? I would think encryption is needed?

Reply:
kms:Decrypt is correct. 

You can find the details here: https://docs.aws.amazon.com/kms/latest/developerguide/services-dynamodb.html

tl;dr This allow Is not about decrypting or encrypting DDB table contents. This permissions is needed to decrypt the "table key". DynamoDB internally manages the encryption/decryption of your data.

Reply:
@carvantes  thanks for the clarification.